test/helper.ts

```diff
@@ -0,0 +1,34 @@
+// This file contains code that we reuse
+// between our tests.
+
+import { build as buildApplication } from "fastify-cli/helper.js";
```

Comment:
I need to add `.js` extension here, I think this is a type issue to fix on the cli package `fastify-cli/helper.js`

Reply:
Ow, maybe we need to fix this upstream in that package

Reply:
Yep, there are several other things I want to update on this package. See https://github.com/fastify/demo/pull/1#discussion_r1644008140

I think I am gonna work on it before continuing the demo.

Reply:
It is the correct way for ESM, `fastify-cli` do not provide exports mapping. So, extension-less will not work.